### DISCUSSION TITLE
Reconnect Inputs

### DISCUSSION THREAD

**Discussion by JamesMolina007** [Foblex/f-flow#83] [General]
Is there a way to reconnect, meaning to change the node only for the input? I know it can be done with the output since I see it in the documentation and have already implemented it. But can it be done with the input?

_In short, I would like to be able to change the connection on both the Input and Output side._

**Comment by siarheihuzarevich** [comment]
Hi! Thanks for your question.

You’re right — currently, only reconnecting from the output side is supported.
But we’re actively working on adding input-side reconnection as well, so that connections can be adjusted from either end.

This feature is planned for an upcoming release — stay tuned, and thanks again for your interest in the library!
